data/json/items/comestibles/med.json

```diff
@@ -564,13 +564,14 @@
         "id" : "disinfectant_powdered",
         "type" : "COMESTIBLE",
         "comestible_type" : "MED",
-        "name" : "powdered disinfectant",
-        "description" : "A powdered form of disinfectant. Could be used as a disinfectant, or mixed with water to become disinfectant",
+        "name" : "cleaning powder",
+        "name-plurarl": "cleaning powder"
```

Comment:
Typo.

Reply:
I literally just fixed that in my last commit.